test/integration/health_check_test.rb

```diff
@@ -0,0 +1,76 @@
+require 'test_helper'
+
+class HealthCheckTest < ActionDispatch::IntegrationTest
+
+  def setup
+    super
+    @headers = { 'Accept': 'application/json' }
+  end
+
+  def test_default_check_passed
+    get("/healthcheck/default.json", headers: @headers)
```

Comment:
- I think it should be `get /healthcheck/default`, headers: ..., as: :json`.
- Wouldn't it be better to use name routes here rather than hardcoding?

Reply:
Unfortunately, this is a OkComputer::Engine inner routes, the only way to get exact route here is to have a hardcoded URL, other way (if we use something like :healthcheck_url) we've get only one default endpoint for overall check.
But changed to as: :json anyway.

Reply:
seems perfect!